test/hotspot/jtreg/compiler/vectorization/runner/ArrayInvariantFillTest.java

```diff
@@ -72,10 +72,10 @@ public ArrayInvariantFillTest() {
     @Test
     @IR(applyIfCPUFeatureOr = {"asimd", "true", "sse2", "true"},
         applyIf = {"OptimizeFill", "false"},
-        counts = {IRNode.REPLICATE_B, ">0"})
+        counts = {IRNode.REPLICATE, ">0"})
```

Comment:
This may be a good time to try and convert `IRNode.REPLICATE` to a `vectorNode`. That would ensure that we still can use the type of the replicate in this test here.

Reply:
Thanks, fixed.